src/base.h

```diff
@@ -0,0 +1,12 @@
+#ifndef SPGLIB_BASE_H
+#define SPGLIB_BASE_H
+
+#include "spglib.h"
+
+#ifdef SPG_TESTING
+#define SPG_API_TEST SPG_API
```

Comment:
Where is `SPG_API` used?

Reply:
It's defined in `spglib.h` and it's used to export/import funtions to/from dll. On unix it only does export